### ISSUE TITLE
Loading the result of a SPARQL CONSTRUCT into another Store object

### ISSUE THREAD

**Issue by milvld** [oxigraph/oxigraph#219]
Hi

As part of an ETL I'm creating some sort of 'input graph' on which I apply some transformations with SPARQL. I'm trying to retrieve the results from those transformations and put them into an 'output graph' (which I want to serialize later on, and send its serialization via Graphstore Protocol to a triple store located elsewhere).

I'm currently using the code below, but I'm wondering whether there's a way to directly load resulting triples into a Store object, rather than first constructing a Quad (which you can use with the `add` method, as opposed to a Triple).

```
pyoxi_input_graph = Store()

pyoxi_input_graph.bulk_load(input=rdf_data_io, mime_type="application/n-triples")

pyoxi_results = list(pyoxi_input_graph.query(query))

for i in range(0,len(pyoxi_results)):
    (s,p,o) = pyoxi_results[i]
    pyoxi_output_graph.add(Quad(s,p,o,DefaultGraph()))
```
So, TL;DR: is there an equivalent to the `add` method to add Triples instead of Quads?

**Comment by Tpt** [comment]
Sorry, it seems I completely missed to answer to this. Here I am a year later...

Sadly, there is no equivalent yet. An option might be to add a `graph_name` option to the `Store.extend` method to automatically convert the `Triple` iterator to a `Quad` iterator.
